### PR TITLE
Changed Mono to Xamarin.Android

### PR DESCRIPTION
**Pull request by TheCodeTraveler**

### CODE REVIEW

File: docs/sdk/push/migration/xamarin-android.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ ms.tgt_pltfrm: android
 
 [!include[](introduction-android.md)]
 
-## 1. Mono version
+## 1. Xamarin.Android version
```
Comment (guperrot) [inline docs/sdk/push/migration/xamarin-android.md]: Target framework version (as per the UI).

File: docs/sdk/push/migration/xamarin-android.md [hunk 2]
```diff
@@ -14,9 +14,9 @@ ms.tgt_pltfrm: android
...
 
-If your Android project does not target Mono framework version 8.1 or higher, you will not be able to update the Push package.
+If your Android project does not target Xamarin.Android version 8.1 or higher, you will not be able to update the Push package.
```
Comment (guperrot) [inline docs/sdk/push/migration/xamarin-android.md]: If the target framework version of your Android application is lower than 8.1, you will not be able to update the Push package.

**Review by guperrot — COMMENTED** [review]
Suggesting an alternative wording

File: docs/sdk/push/migration/xamarin-android.md (hunk 1 above)
Comment (TheCodeTraveler) [reply docs/sdk/push/migration/xamarin-android.md]: image
👍 Good call with `Target Framework`. That's what the IDE calls it.

**Review by guperrot — APPROVED** [review]

**Pull request merged**